Add eslint, eslint_d linters

sirreal opened the pull request.

BBboy01 commented on README.md:
```diff
@@ -36,6 +36,7 @@
 
 - [x] [clang-tidy](https://clang.llvm.org/extra/clang-tidy/)
 - [ ] [codespell](https://github.com/codespell-project/codespell)
+- [ ] [eslint_d](https://www.npmjs.com/package/eslint_d)
```
```suggestion
- [ ] [eslint_d](https://github.com/mantoni/eslint_d.js)
```

github URL maybe better?

xiaoshihou514 commented:
@sirreal Could you kindly rebase? Or use the sync button via webui.

sirreal commented:
Rebased 👍

sirreal commented:
I added `find`, added this to the linter init, and added `eslint`. Eslint and eslint_d use nearly the same config, if there's a better way to share that config let me know.
